src/webapi/blob.rs

```diff
@@ -31,6 +31,22 @@ pub trait IBlob: ReferenceType {
             Some( mime )
         }
     }
+
+    /// Create a new `Blob` object containing the data in the specified range of bytes of the
+    /// source `Blob`.
+    /// 
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Blob/slice)
+    /// https://w3c.github.io/FileAPI/#ref-for-dfn-slice
```

Comment:
The link to the specs should not be part of the docs. (Change `///` into `//` for this line.)